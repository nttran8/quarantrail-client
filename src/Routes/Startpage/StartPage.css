.startpage {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  position: relative;
}

.startpage-character {
  width: 150px;
}
