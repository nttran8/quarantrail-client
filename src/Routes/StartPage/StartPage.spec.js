import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import StartPage from './StartPage'

describe(`StartPage Route`, () => {

  it('renders by default', () => {
    const wrapper = shallow(<StartPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})