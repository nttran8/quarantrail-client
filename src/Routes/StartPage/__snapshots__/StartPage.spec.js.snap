// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartPage Route renders by default 1`] = `
<div
  className="startPage"
>
  <Character
    selectCharacter={true}
  />
  <form
    className="nameForm"
    onSubmit={[Function]}
  >
    <div
      className="startPage-inputbox"
    >
      <label
        htmlFor="name"
      >
        Name 
      </label>
      <input
        id="name"
        onChange={[Function]}
        required={true}
        type="text"
      />
    </div>
    <button
      className="introButton"
    >
      Submit
    </button>
  </form>
  <Music
    song="8bitmenu.mp3"
  />
</div>
`;
